app/components/onboarding/OnboardingDashboard.tsx

```diff
@@ -44,6 +51,7 @@ const initialValues: UserFormData = {
 const OnboardingDashboard = ({ dict }: OnboardingProps) => {
   const [onboardingStep, setOnboardingStep] = useState<number>(0);
   const [values, setValues] = useState<UserFormData>(initialValues);
+  const [siteToView, setSiteToView] = useState<number>(-1);
```

Comment:
I think it is more clear if null or undefined is used to represent that no site is selected at the moment